ludwig/features/number_feature.py

```diff
@@ -230,21 +231,6 @@ def preprocessing_defaults():
             "normalization": None,
         }
 
-    @staticmethod
-    def preprocessing_schema():
-        return {
-            "missing_value_strategy": {
-                "type": "string",
-                "enum": MISSING_VALUE_STRATEGY_OPTIONS,
-            },
-            "fill_value": {"type": "number"},
-            "computed_fill_value": {"type": "number"},
-            "normalization": {
-                "type": ["string", "null"],
-                "enum": list(numeric_transformation_registry.keys()),
```

Comment:
I believe this list gets duplicated (once here, once in the preprocessing schema), which is unfortunate. Unrelated to this PR but another indication we should do a wider registry refactor at some point.